publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/grant/CuratorPermissionStrategy.java

```diff
@@ -24,7 +24,7 @@ public CuratorPermissionStrategy(Publication publication, UserInstance userInsta
 
     @Override
     public boolean allowsAction(PublicationOperation permission) {
-        if (!canManageStandardResources() || !userRelatesToPublication()) {
+        if (!canManageStandardResources() || !userRelatesToPublication() || userInstance.isExternalClient()) {
             return false;
```

Comment:
Is there a simpler way to do this? Seems wrong to include a negative isExternalClient here. Is there something else we can check? Can we do a DenyStrategy?

Reply:
Was thinking about it, but could not figure it out. I decided that the easiest and best solution is this for now and later make sure external clients have scopes and use the normal policies. Then we could have a simple deny for external if customer id is not matching. It you have other ideas, let me know :-)

Reply:
I will remove this check for now!